src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier;
+    private Random random = new Random();
+    private final int magician = 10;
+    private final int lownum = 1;
+
+    public FigureSupplier(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Square getRandomSquare() {
```

Comment:
Private ones (methods, classes) should be after public ones.
```suggestion
    private Square getRandomSquare() {
```